templates/python36builddc.json

```diff
@@ -0,0 +1,318 @@
+{
```

Comment:
Looks like this is missing recent template changes in a diff against pythonbuilddc.json

addition of lookupPolicy
addition of kube scheduler param and env var
addition application_name env var